src/wallet/wallet2.cpp

```diff
@@ -15220,11 +15218,11 @@ std::vector<std::pair<uint64_t, uint64_t>> wallet2::estimate_backlog(const std::
         priority_weight_max += i.weight;
     }
 
-    uint64_t nblocks_min = priority_weight_min / full_reward_zone;
-    uint64_t nblocks_max = priority_weight_max / full_reward_zone;
-    MDEBUG("estimate_backlog: priority_weight " << priority_weight_min << " - " << priority_weight_max << " for "
-        << our_fee_byte_min << " - " << our_fee_byte_max << " piconero byte fee, "
-        << nblocks_min << " - " << nblocks_max << " blocks at block weight " << full_reward_zone);
+    uint64_t nblocks_max = priority_weight_min / full_reward_zone;
+    uint64_t nblocks_min = priority_weight_max / full_reward_zone;
+    MDEBUG("estimate_backlog: given a block weight of " << full_reward_zone << " you will need to wait "
```

Comment:
Might be advisable to add an assert here to assure this kind of thing doesn't happen again.

Reply:
Good idea, addressed. Pushed the squashed commit.